src/components/Todo/Todo.tsx

```diff
@@ -67,14 +75,37 @@ function TodoApp(props: TodoAppProps) {
         const item = todoItems.find((item) => item.uuid === itemKey);
         if (item) {
           currItems.push(item);
+
         }
         return currItems;
       },
       []
     );
     setItems([...updatedItems, ...completedItems]);
+
+
+  };
+  const handleKeyPress = (event: any) => {
+    if (`${event.key}` == 'z' && `${event.key.control}`) {
```

Comment:
The name of the function is misleading as keypress is another event. Also please use type for event instead of any.

Reply:
i used " keyboardEvent" but throw error and shortcuts don't work
 Type '(event: KeyboardEvent) => void' is not assignable to type 'EventListener'.
        Types of parameters 'event' and 'evt' are incompatible.
          Type 'Event' is missing the following properties from type 'KeyboardEvent<Element>': altKey, charCode, ctrlKey, code, and 15 more

Reply:
okay. You can ignore the type issue. Please fix the undo behavier for a textfield and outside text field as mentioned in this comment.
https://github.com/upnotes-io/react-todo/pull/47#issuecomment-1292023555

Reply:
@kamalkishor1991   while focus on item undo work on text otherwise on multiple items
but i have problem when i passing props from todo to item 
"i am not good with typscript"